### DISCUSSION TITLE
Possible to do multiple instances for multiple location

### DISCUSSION THREAD

**Discussion by jbteamhome** [FelixdelasPozas/TrayWeather#112] [Ideas]
Hello,

This is a great app. I'm curious if its possible to run multiple instances so I can track multiple locations?

Thank you

**Comment by FelixdelasPozas** [comment]
Hi @jbteamhome .

Right now is not possible to run multiple instances simultaneously. The scope of the application is to have the weather forecast for the computer location in an easy-to-see way and just for that location. For that Geo-location is used to automatically get the coordinates of the IP address if needed, and the "roaming mode" is for when the computer is "on the move". 
While it could be possible to modify it and have the program ask for the weather of multiple coordinates (and I think that the limits of the free OpenWeatherMap API, 60 calls per minute, are more than enough), it is not the intention of the program.

Regards, 
Félix.
